WooCommerce/WooCommerceUITests/Tests/PaymentsTests.swift

```diff
@@ -19,4 +19,11 @@ final class PaymentsTests: XCTestCase {
             .tapChipperManual()
             .verifyChipperManualLoadedOnWebView()
     }
+
+    func test_load_payments_universal_link() throws {
```

Comment:
Nitpick: _possibly_ this name is more straightforward:

```suggestion
    func test_univeral_link_loads_payments() throws {
```

Reply:
in WCiOS, the tests (well, except the login-related ones) are named this way: `{the word "test"}-{verb to describe the test}-{test description}` for naming consistency, i prefer to keep it this way

Reply:
I did not realize there's a consistent naming style for the tests in WCiOS. Then it makes sense 👍